cli/logger/color_logger.go

```diff
@@ -29,8 +30,23 @@ func NewColorLoggerFactory() *ColorLoggerFactory {
 	}
 }
 
-// Create implements logger.Factory.Create.
-func (c *ColorLoggerFactory) Create(name string) logger.Logger {
+// CreateContainerLogger implements logger.Factory.CreateContainerLogger.
+func (c *ColorLoggerFactory) CreateContainerLogger(name string) logger.Logger {
+	return c.create(name)
```

Comment:
ColorLoggerFactory does not also implement the LoggerInterface, its doing some work to wrap the factory via a logger interface